docs/docs/configuration/external-mappings.mdx

```diff
@@ -73,6 +73,38 @@ it will use the provided implementation by the `BananaMapper`.
 Used mappers themselves can be Mapperly backed classes.  
 The [`AutoUserMappings`](./user-implemented-methods.mdx) value also applies to the usage of external mappers.
 
+## Globally available external mappers
```

Comment:
Not sure about the term "globally" here... It is still limited to the scope of the assembly...